helmfile/values/s3-exporter.yaml.gotmpl

```diff
@@ -0,0 +1,15 @@
+s3:
```

Comment:
I think `forcePathStyle` is missing from here, or should we just always have it set to `true`?

Reply:
Good point, I can add it :+1: